text/machine_comprehension/gpt-2/GPT2-export.py

```diff
@@ -13,6 +14,22 @@
 data_dir = 'test_data_set_0'
 
 
+def f(t):
```

Comment:
I suggest we provide more descriptive names for these methods.

Reply:
Seconded. Arguments, also.